client/src/components/common/names-capture.vue

```diff
@@ -41,12 +41,12 @@
         </v-row>
       </template>
 
-      <v-row :class="editMode ? '' : 'mt-3' ">
+      <v-row :class="editMode ? '' : isAssumedName ? 'mt-3' : 'mt-9' ">
         <v-col cols="12"
                class="h4 mb-3 ml-n1"
                v-if="editMode">Name Choices
         </v-col>
-        <v-col cols="12" v-if="!editMode && isAssumedName" class="text-body-3">
+        <v-col cols="12" v-if="!editMode && isAssumedName" class="text-body-3 mb-4">
```

Comment:
I discourage bottom whitespace because this element shouldn't assume what's below it. Instead, the element below should specify top whitespace (because it knows what's above it). I guess this is a best practise instead of a rule but I have found that it really simplifies layouts.